Core/StringCache.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  StringCache.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public class StringCache {
```

Comment:
StringCache sounds generic and implies that any string can be stored here. But that's not the case as the strings are specifically shared with "group.com.duckduckgo.contentblocker". Can we rename this class to better express that it's a content blocker string cache?